BikeRental/Models/Bike.cs

```diff
@@ -4,17 +4,21 @@ namespace BikeRental.Models
 {
     public class Bike : BaseEntity
     {
-        public int OwningStoreId { get; set; }
-
+        public Bike()
+        {
+            Reservations = new HashSet<Reservation>();
+        }
         public int FrameSize { get; set; }
-        public decimal Surcharge { get; set; }
-        public decimal Price { get; set; }
-        public bool ElectricMotor { get; set; }
-        public bool AllTerrain { get; set; }
-        public bool Available { get; set; }
+        public bool? ElectricMotor { get; set; }
+        public bool? AllTerrainSuspension { get; set; }
         public string BikeStyle { get; set; }
+        public int BikeId { get; set; }
```

Comment:
All these models are inheriting BaseEntity, which provides an ID and CreatedAt property
Can remove the BikeID

Reply:
I cant remove the BikeID because its referenced in the context class